src/omv/modules/py_imageio.c

```diff
@@ -186,7 +186,10 @@ mp_obj_t py_imageio_read(uint n_args, const mp_obj_t *args, mp_map_t *kw_args)
         stream->ms = ms;
         read_long(fp, (uint32_t *) &image.w);
         read_long(fp, (uint32_t *) &image.h);
-        read_long(fp, (uint32_t *) &image.bpp);
+        uint32_t bpp;
+        read_long(fp, (uint32_t *) &bpp);
+        // TODO: workaround for backwards compatibility until the stream version is updated.
+        image.pixfmt = (bpp == 1) ? PIXFORMAT_GRAYSCALE : PIXFORMAT_RGB565;
```

Comment:
It could be 0, 1, 2, 3, or Jpeg...

Reply:
How do I know if it's bayer or grayscale ? The stream file format must be updated. That's why I'm doing this, everything was built on the assumption that BPP is unique.